calcpay.py

```diff
@@ -94,6 +125,23 @@ def calculate_pay(employee_id: int, start_date: str, end_date:str, df_employee,
     return df_wage.sum()
 
 
+    # Dates inputted by user to calculate
+    first_date = pd.to_datetime(start_date, format="%Y-%m-%d")
+    last_date = pd.to_datetime(end_date, format="%Y-%m-%d")
+    
+    # Checks if the worked dates fall between the selected start and end date. Also check if employee id matches
+    # NOTE: Using binary (&) operator to make comparison between datetime.timestamp and datetime64
+    #  - Returns a parallel dataframe with true/false that show if the row in df_merged matches
+    mask = ((df_merged.Date >= first_date) & (df_merged.Date <= last_date)) & (df_merged.EmployeeId == employee_id)
+    print(mask)
```

Comment:
remove the print statements unless they are a part of the program

Reply:
okay